filebeat/module/apache2/_meta/docs.asciidoc

```diff
@@ -41,6 +41,9 @@ file to override the default paths for Apache2 access and error logs:
   access:
     enabled: true
     var.paths: ["/path/to/log/apache2/access.log*"]
+  ssl:
```

Comment:
We should move this below `error` to be alphabetical. Any specific reason you put it in between?